docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Setup a Highly Available Etcd Cluster With Kubeadm
+---
+
+{% capture overview %}
+Kubeadm by default will create a self-hosted etcd cluster of one member. This is
```

Comment:
It's not self-hosted as per the definition of "self-hosting" we have in the community. I'd say it like this:

```
By default, kubeadm will run one etcd instance in a Static Pod ran by the kubelet on the master. This is not a highly-available setup in case your master with that one etcd replica goes unavailable.
```